CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,45 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
```

Comment:
I think this is also the responsibility of the community itself and not just the maintainers, but perhaps that is implied by the pledge above?

Reply:
We should make it more explicit that the code of conduct guidelines apply to entire community and all the individuals within the community know that they have to abide by the rules. 

As it is common across all the projects, it is more appropriate for Technical Committee to be responsible for it in its entirety (creation, clarification, followup, taking actions, etc) rather than project maintainers. Again, this is another item to have a discussion within TC.

Reply:
Should it be a responsibility of the owners to raise any issues, rather than doing local things?

Reply:
I think as a good citizens, everyone who is a contributor to the community is responsible to raise issues/bring them to attention of TC or the people who are endorsed by TC to work on code of conduct issues, preferably via that private mail alias without publicizing it.

The owners should also follow the same process to raise issues rather than taking action themselves.

TC is responsible to do periodic code of conduct reminders to entire community on a quarterly basis for example rather than just adding the file there and hoping for the best. This will help raise the awareness for the members who joined recently and also make others remember it.